### PR TITLE
Zjr/site migration

### PR DESCRIPTION
**Pull request by Zach-Bob**
Create basic migration and basic schema for the site table. This moves us towards our goal of allowing a user to add and remove sites from the system.
A dependency problem is also fixed - as of version 3.0, ecto requires the ecto_sql dependency for `Ecto.Adapters.Postgres` so that was added to mix.exs

### CODE REVIEW

File: lib/situation_room/sites.ex
Comment (cpjolicoeur) [inline lib/situation_room/sites.ex:1]: In Elixir, we typically name the models for schema objects in the Singluar.  So this should be `SituationRoom.Site`

File: lib/situation_room/sites.ex
Comment (cpjolicoeur) [inline lib/situation_room/sites.ex:4]: See note on previous commit about table names in the plural

File: mix.exs
Comment (cpjolicoeur) [inline mix.exs:25]: Squash this commit back into the first commit since that is where this line was originally removed

**Review by cpjolicoeur — CHANGES_REQUESTED** [review]

File: lib/situation_room/sites.ex
Comment (Zach-Bob) [reply lib/situation_room/sites.ex:1]: I have that commit in PR 33. Should I cherry-pick that commit and bring it in this PR or is it fine to leave it in the next PR? As it stands currently, PR 33 should be ready to merge right after this one.

**Comment by Zach-Bob** [comment]
This PR will be encompassed into the next PR

**Pull request closed without merging**
